<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sppeculate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sppeculate.qrc">
    <normaloff>:/share/icons/app-icon.svg</normaloff>:/share/icons/app-icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>803</width>
           <height>256</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="infoEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(89, 89, 89, 220);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFolder"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomDefault"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionPrevItem"/>
   <addaction name="actionNextItem"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/gnome-shutdown.svg</normaloff>:/share/icons/gnome-shutdown.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/folder-new.svg</normaloff>:/share/icons/folder-new.svg</iconset>
   </property>
   <property name="text">
    <string>Open &amp;folder</string>
   </property>
   <property name="toolTip">
    <string>Open image folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/help-browser.svg</normaloff>:/share/icons/help-browser.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show about box</string>
   </property>
  </action>
  <action name="actionNextItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/go-next.svg</normaloff>:/share/icons/go-next.svg</iconset>
   </property>
   <property name="text">
    <string>NextItem</string>
   </property>
   <property name="toolTip">
    <string>Next picture</string>
   </property>
  </action>
  <action name="actionPrevItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/go-previous.svg</normaloff>:/share/icons/go-previous.svg</iconset>
   </property>
   <property name="text">
    <string>PrevItem</string>
   </property>
   <property name="toolTip">
    <string>Previous picture</string>
   </property>
  </action>
  <action name="actionZoomDefault">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/zoom-original.svg</normaloff>:/share/icons/zoom-original.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomDefault</string>
   </property>
   <property name="toolTip">
    <string>Zoom to deault level</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/zoom-out.svg</normaloff>:/share/icons/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sppeculate.qrc">
     <normaloff>:/share/icons/zoom-in.svg</normaloff>:/share/icons/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sppeculate.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
